hack/make/.go-autogen

```diff
@@ -8,11 +8,12 @@ cat > autogen/dockerversion/dockerversion.go <<EOF
 package dockerversion
 
 var (
-	GITCOMMIT string = "$GITCOMMIT"
-	VERSION   string = "$VERSION"
+	GITCOMMIT    string = "$GITCOMMIT"
+	VERSION      string = "$VERSION"
```

Comment:
The only thing there that worries me is `req.Header.Set("User-Agent", "Docker-Client/"+dockerversion.VERSION)` because I've heard rumors that the Hub team reads this to change the behavior of the Hub based on which version of Docker is making the request. :smile: